addons/mail/models/mail_activity_mixin.py

```diff
@@ -211,6 +211,8 @@ def _search_activity_date_deadline(self, operator, operand):
 
     @api.model
     def _search_activity_user_id(self, operator, operand):
+        if operator == '=' and not operand:
```

Comment:
Did you check if there is an unit test covering the "= False" somewhere ? If not, could be worth adding it. If yes, heu voila.

Reply:
Added a small test